notebooks/ROS2_Turtlesim.ipynb

```diff
@@ -0,0 +1,8029 @@
+{
```

Comment:
Based on the commit messages, it does not seem like this file and `turtle_sim.py` were meant to be included in this PR. Please remove if necessary.